docs/www/autotune.md

```diff
@@ -6,7 +6,7 @@ order: 1
 # Designed for performance
```

Comment:
`Update issue vectorizedio#2879: removing error throwing if less than 1GB`

* the general format of the commit message is usually `subsystem: subject` so in this case it would be something like `syscheck: allow startup with less than 1 GB per core`.
* the commit message then should have a short description of why the change is being made. that can probably be taken from the corresponding issue 2879

Reply:
[d1c5f2f](https://github.com/vectorizedio/redpanda/pull/3028/commits/d1c5f2fbec1584cf159a21f0cbe1e3c3a73ed9d3)